docs/entity_air_pollutants.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Air Pollutants Entity
+sidebar_label: Air Pollutants
+---
+
+## Properties
+
+> Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
+
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| state | string | **Required** | The current PM 2.5 level.
+| particulate_matter_2_5 | float | **Required** | The particulate matter 2.5 (<= 2.5 μm) level.
+| particulate_matter_10 | float | **Required** | The particulate matter 10 (<= 10 μm) level.
+| temperature | float | `None` | The current temperature in °C or °F.
```

Comment:
These should be published as a sensor with temp class and not be part of this entity. There should only be 1 place where something will be stored.

Same goes for humidity

Reply:
Thanks